### ISSUE TITLE
An add repeatedly interrupts the WAMU stream

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tripodtr/npr-android-app#100]
```
What steps will reproduce the problem?
1. Load WAMU's HD stream in the app
2. Listen for a minute or so
3. An add interrupts the stream

What is the expected output? What do you see instead?

The audio should play, uninterrupted

What version of the product are you using? On what operating system?

Latest version of the app. HTC Incredible with OS 2.2

Please provide any additional information below.

An ad for American University's School of Communications repeatedly interrupts 
the audio stream
```

Original issue reported on code.google.com by `jlucas....@gmail.com` on 25 Jul 2011 at 7:43

**Comment by GoogleCodeExporter** [comment]
```
Another listener reported essentially the same error for WNYC:

"Concerning WNYC: A sponsor announcement cuts into the broadcast at irregular 
times, other than the normal station breaks. This happens on all streaming 
apps, not only NPR. It happens rearly on other stations but quite.regularly on 
WNYC."

I was able to duplicate the error with WNYC as well. In this case it is an ad 
for the NYU school for continuing studies.
```

Original comment by `jlucas....@gmail.com` on 25 Jul 2011 at 7:59

**Comment by GoogleCodeExporter** [comment]
```
Another listener reported this issue for WHYY and I was able to confirm:

"body : I am not in range to listen to NPR on the radio, so I use the Android 
app to listen to the closest station, 
WHYY in Philadelphia.  Whenever I try to listen Terry Gross comes on to remind 
me to donate, over and over again.  I 
understand that NPR must solicit donations to keep in the black, and I donate 
what I can, and I also understand that 
the streaming service is an additional expense, however, the solicitation are 
getting in the way of the service.  
Even when I am in good cell service, only a few seconds of Terry Gross's 
message makes it through, and almost always 
cuts off at the exact same time.  The app will try to stream a few times and 
then give up; each time I hear a little 
bit of the solicitation.  I have never before been tempted to throw my phone, 
but this is infuriating.  I'm not 
asking you to stop using solicitation every time some one connects to the live 
stream, but please don't make me 
listen to it 30 times and never let me actually listen to the stream.  This 
morning, it finally went all the way 
through the message after several minutes of it failing, and then it started 
over again.  Please, either buffer the 
whole message before starting the stream, limit the number of times it tries, 
or something.  It is incredibly 
irritating to be asked for money over and over again for a system that doesn't 
work because it's too intent on 
asking for money."
```

Original comment by `jlucas....@gmail.com` on 29 Jul 2011 at 5:53

**Comment by GoogleCodeExporter** [comment]
```
Could not reproduce this on a Nexus One. Will try grabbing a log from Justin 
tomorrow.
```

Original comment by `jpenn...@gmail.com` on 2 Aug 2011 at 8:26

**Comment by GoogleCodeExporter** [comment]
```
Is this still an issue?
```

Original comment by `jpenn...@gmail.com` on 10 Jan 2012 at 7:29

**Comment by GoogleCodeExporter** [comment]
Original comment by `jpenn...@gmail.com` on 13 Feb 2012 at 8:28
- Changed state: **WontFix**
